test/metabase/query_processor_test/date_bucketing_test.clj

```diff
@@ -1029,6 +1032,7 @@
              (mt/formatted-rows [int]
                (mt/run-mbql-query checkins
                  {:aggregation [[:count]]
+                  ;; TODO: somehow this is getting turned into timestamp '2019-01-16 00:00 UTC'
```

Comment:
Is this TODO still true? If it is, it might be worth clarifying that it's getting turned into something for Presto JDBC and not generally.

Reply:
Nope, removed now.